packages/patternfly-react/src/components/SessionTimeout/SessionTimeout.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Timer from '../../common/Timer';
+
+import { Modal, Button, Col, Icon } from '../../index';
+
+export const SessionTimeoutModal = props => (
+  <Modal show={props.show}>
+    <Modal.Body className="session-timeout-body">
+      <Col className="session-timeout-warning-icon" md={2}>
+        <Icon type="pf" name="warning-triangle-o" />
+      </Col>
+      <Col className="session-timeout-message" md={10}>
+        {props.children}
+      </Col>
+    </Modal.Body>
+    <Modal.Footer>
+      <Button bsStyle="default" onClick={props.logoutFnc}>
+        {props.logoutText}
+      </Button>
+      <Button bsStyle="primary" onClick={props.continueFnc}>
+        {props.continueText}
+      </Button>
+    </Modal.Footer>
+  </Modal>
+);
```

Comment:
We should use the MessageDialog here so the session timeout dialog is consistent with its layout.